src/nfa/operations.cc

```diff
@@ -34,9 +37,14 @@ using namespace Mata::Nfa;
 using Mata::Symbol;
 using Mata::BoolVector;
 
+// Vector of bool is probably slower than vector of char (from some little experiment).
+// Just BoolArray would be better?
+// It would make sense to make this type name global, it is type that we use globally to represent bool vectors. ?
 using StateBoolArray = std::vector<bool>; ///< Bool array for states in the automaton.
 
```

Comment:
I think `BoolArray` has potential to be faster. We can try and test the performance when using `BoolArray`.

Reply:
n the second line I meant changing the name, StateBoolArray -> BoolArray, because it will be the same thing ofr states, symbols, transitions, and whatever else, so I would remove the "State" from the name.

Reply:
yes

Reply:
That can be done, too.